### PR TITLE
Perfecting custom endpoint mechanism 

### LINKED ISSUE #219: Perfect the custom endpoint mechanism for Hydrus.

### PR DESCRIPTION
**Pull request by vaibhavchellani**
Fixes #219 

### Checklist
- [X] My branch is up-to-date with upstream/develop branch.
- [X] Everything works and tested for Python 3.5.2 and above.

### Description

PR modifes the "@id" and "@context" according to custom path if provided 
### Test Logs

screen shot 2018-06-18 at 1 28 30 pm

### CODE REVIEW

**Comment by vaibhavchellani** [comment]
@xadahiya let me know the changes to be made

File: hydrus/data/crud.py [hunk 1]
```diff
@@ -149,12 +152,12 @@ def insert(object_: Dict[str, Any], session: scoped_session, id_: Optional[int]
                 instance_id = insert(object_[prop_name], session=session)
                 instance_object = session.query(Instance).filter(
                     Instance.id == instance_id).one()
-
                 if property_.type_ == "PROPERTY" or property_.type_ == "INSTANCE":
                     property_.type_ = "INSTANCE"
                     session.add(property_)
                     triple = GraphIII(
                         subject=instance.id, predicate=property_.id, object_=instance_object.id)
+                    print(triple)
```
Comment (xadahiya) [inline hydrus/data/crud.py]: Remove print statements.

Comment (vaibhavchellani) [reply hydrus/data/crud.py]: oh left that one

**Comment by vaibhavchellani** [comment]
@xadahiya anything else you can think of which should be included in this PR ?

**Pull request merged**